test/jasmine/tests/click_test.js

```diff
@@ -418,8 +418,8 @@ describe('Test click interactions:', function() {
             var node = document.querySelector('rect.nwdrag');
             var pos = getRectCenter(node);
 
-            expect(node.classList[0]).toBe('drag');
-            expect(node.classList[1]).toBe('nwdrag');
+            expect(node.classList[1]).toBe('drag');
+            expect(node.classList[0]).toBe('nwdrag');
```

Comment:
OK as is, but this seems ripe for a custom matcher like:
```
expect(node).toBeClassed(['drag', 'nwdrag', 'cursor-nw-resize']);
```
That would be order-independent but also verify these are the *only* classes applied.

Reply:
Good call.

`cartesian_test.js` uses assertions like:

```js
expect(fills[0].classList.contains('js-tozero')).toBe(true)
```

which could work here too, but yeah checking adding a custom assertion that test multiple class names at once should be worthwhile.

Reply:
done in b974eab